ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/api/UsesEditorConfigProperties.kt

```diff
@@ -297,6 +293,7 @@ public object DefaultEditorConfigProperties : UsesEditorConfigProperties {
         UsesEditorConfigProperties.EditorConfigProperty(
             type = PropertyType.max_line_length,
             defaultValue = -1,
+            defaultAndroidValue = 100,
```

Comment:
Should this be added somewhere to the CHANGELOG? Or has it always been the case?

Reply:
Yes, it has always been there (inside the propertyMapper). I had to rewrite this due to the new way of resolving default editor config properties.